lambdas/service/app.py

```diff
@@ -219,42 +219,110 @@ def openapi_raw():
 
 health_spec = {
     'responses': {
-        '200': {
-            'description': '200 response',
+        f'{200 if up else 503}': {
+            'description': f'Service component is {"" if up else "un"}reachable.',
             'content': {
                 'application/json': {
-                    'schema': {'$ref': '#/components/schemas/Empty'}
+                    'schema': schema.object(
+                        additional_properties=schema.object(
+                            additional_properties=True,
+                            up=schema.enum(up)
+                        ),
+                        up=schema.enum(up)
+                    )
                 }
             }
-        }
+        } for up in [True, False]
     },
     'tags': ['Auxiliary']
 }
 
+health_service_keys = '`elasticsearch` indicates whether service can establish a connection with Elasticsearch, ' \
+                      'ensuring that indexed documents can be access through Elasticsearch. ' \
+                      '`api_endpoints` shows a list of service API endpoints being requested by service, ' \
+                      'confirming that a client making a similar request will obtain a OK response.'
+
+health_all_keys = '`queues` lists queues utilized by indexer to process transactions and index documents into ' \
```

Comment:
Use triple quoted strings and `format_description` from https://github.com/DataBiosphere/azul/pull/1435/files. Base your PR branch on that PR's branch to get that function. Use a bulleted list with markdown.

Then again, maybe this should be more formally spec-ed out in the form of a JSONSchema for the response body. The schema doesn't have to be 100% complete. It merely needs to list all the top-level keys along with a description. You may have had that already and I asked you to remove it. Please reach out to me if that is the case so we can discuss.